app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1671,15 +1665,22 @@ table li .account_details {
 }
 
 .item-show-price {
-  border: 1px solid red;
   text-align: center;
   font-size: 52px;
   margin: 20px 0 10px 0;
   font-family: 'Cardinal', Arial, sans-serif;
 }
 
+.item-description {
+  font-family: 'Cardinal', Arial, sans-serif;
```

Comment:
Properties should be ordered background, font-family, font-size, margin-top, opacity, padding